libavpipe/src/avpipe_xc.c

```diff
@@ -4186,6 +4186,20 @@ avpipe_probe(
             ((float)stream_probes_ptr->duration_ts)/stream_probes_ptr->time_base.den)
             probe->container_info.duration =
                 ((float)stream_probes_ptr->duration_ts)/stream_probes_ptr->time_base.den;
+
+        av_dict_copy(&stream_probes_ptr->tags, s->metadata, 0);
+
+        for (int i = 0; i < s->nb_side_data; i++) {
+            const AVPacketSideData *sd = &s->side_data[i];
+            switch (sd->type) {
+                case AV_PKT_DATA_DISPLAYMATRIX:
+                    stream_probes_ptr->side_data.display_matrix.rotation = av_display_rotation_get((int32_t *)sd->data);
```

Comment:
It might be helpful to convert this to CW rotation, as that is what we are using by default in Reza's PR #46. Right now this is CCW rotation, represented from -180 to 180. Can do something like:

```c
double rot;
rot = av_display_rotation_get((int32_t *)sd->data);
// Convert to [0, 360] range
rot = rot >= 0 ? rot : 360.0 + rot;
// Convert from CCW to CW
rot = 360 - rot;
stream_probes_ptr->side_data.display_matrix.rotation = rot;
```

Reply:
Ok I added a field `rotation_cw` (and preserved the original as is)